modules/base/main.tf

```diff
@@ -48,7 +48,7 @@ module "k8s" {
 # ######################################################################
 module "res_defs" {
   source           = "../htc_res_defs"
-  k8s_cluster_name = var.gke_cluster_name
+  k8s_cluster_name = module.k8s.cluster_name
```

Comment:
This is needed to ensure the cluster isn't destroyed before the cluster resource definition is to allow Humanitec to properly cleanup all apps depending on this cluster.